misc-tools/import-contest

```diff
@@ -189,8 +189,11 @@ if os.path.exists('problems.yaml') or os.path.exists('problems.json') or os.path
         if not problems_imported:
             if os.path.exists('problems.yaml'):
                 problems_file = 'problems.yaml'
-            else:
+            elif os.path.exists('problems.json'):
                 problems_file = 'problems.json'
+            else:
+                print("Neither problems.{yaml,json} found.")
```

Comment:
This is basically where it crashed for me as reported in Slack. The condition in line 177 also allows `problemset.yaml`.

Reply:
Should I take it out so it can be addressed in another PR as this will now be discussed during CLICS?

Reply:
I will fix this